librosa/display.py

```diff
@@ -83,6 +84,8 @@
     "AdaptiveWaveplot",
 ]
 
+# mypy: disable-error-code="attr-defined"
```

Comment:
no hope of getting matplotlib variable types here, so we'll just disable this check